deploy/charts/rook-ceph-cluster/templates/cephcluster.yaml

```diff
@@ -15,5 +15,7 @@ spec:
     externalMgrPrometheusPort: {{ toYaml .Values.monitoring.externalMgrPrometheusPort }}
 {{- end }}
 {{- end }}
-
-{{ toYaml .Values.cephClusterSpec | indent 2 }}
+{{- /*
+This function is defined in the helper templates (_helpers.tpl).
+*/ }}
```

Comment:
I think the whitespace might be causing the helm chart failures in the CI. 
```suggestion
*/}}
```

Reply:
Or perhaps remove the `/*` delimiters?
```
Error:  templates/: parse error at (rook-ceph-cluster/templates/cephcluster.yaml:18): comment ends before closing delimiter
```